packages/ui/src/lib/table/Table.svelte

```diff
@@ -270,7 +270,7 @@ function toggleChildren(name: string | undefined): void {
               {#if column.info.renderer}
                 <svelte:component
                   this="{column.info.renderer}"
-                  object="{column.info.renderMapping?.(object) ?? object}"
+                  object="{column.info.renderMapping ? column.info.renderMapping(object) : object}"
```

Comment:
Please apply the identical fix to line 310 (child rows).

Reply:
I applied the fix to the children too! Thanks for catch